eth-connector-tests/src/connector.rs

```diff
@@ -311,6 +311,93 @@ async fn test_ft_transfer_call_without_message() -> anyhow::Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn test_ft_transfer_call_user_message() {
+    let contract = TestContract::new().await.unwrap();
+    contract.call_deposit_eth_to_near().await.unwrap();
+
+    let user_acc = contract.create_sub_account("eth_recipient").await.unwrap();
+    let receiver_id = contract.contract.id();
+    let balance = contract
+        .get_eth_on_near_balance(user_acc.id())
+        .await
+        .unwrap();
+    assert_eq!(balance.0, DEPOSITED_AMOUNT - DEPOSITED_FEE);
+
+    let balance = contract
+        .get_eth_on_near_balance(contract.contract.id())
+        .await
+        .unwrap();
+    assert_eq!(balance.0, DEPOSITED_FEE);
+
+    let transfer_amount: U128 = 50.into();
+    let memo: Option<String> = None;
+    let message = "";
+    // Send to Aurora contract with wrong message should failed
```

Comment:
This comment does not align with the asserts below. It looks like this first call is a transfer to a non-engine, then the second call is to an engine (the receiver is added as a known engine in between), and it is the second call that fails.